src/views/product-tutorials-view/components/sitemap/index.tsx

```diff
@@ -35,7 +35,7 @@ export function ProductTutorialsSitemap({
 				</Heading>
 			</div>
 			<ul className={s.collectionListRoot}>
-				{collections.map((collection: ClientCollection) => (
+				{collections.map((collection: SitemapCollection) => (
```

Comment:
Sitemap already had data nicely formatted server-side, from #420. This fixes up some related types.

Reply:
nice catch